.kitchen.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Google LLC
```

Comment:
Why is this `.kitchen.yml` checked into the top-level directory?  Why not in the `integration_tests` directory?

Reply:
`kitchen.yml` needs to be in the root to do top-level checks and to configure what hardware you need to run the tests on; how to provision the machine; and which combinations of test suites to exercise on the hardware.